citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -133,7 +133,7 @@ protected void handleExportRequest(HttpServletRequest req, HttpServletResponse r
 			Reference ref, String format, String subtype) 
 			throws EntityNotDefinedException, EntityAccessOverloadException, EntityPermissionException 
 	{
-		if(! ContentHostingService.allowGetResource(req.getParameter("resourceId")))
+		if(! ContentHostingService.allowGetResource(req.getParameter("contentCollectionId")))
```

Comment:
It seems strange todo a permission check against one request parameter but then do the export based on another parameter.